TensorLib.lean

```diff
@@ -1,3 +1,5 @@
 -- This module serves as the root of the `TensorLib` library.
 -- Import modules here that should be built as part of the library.
-import TensorLib.Basic
\ No newline at end of file
+import TensorLib.Basic
+import TensorLib.TensorElement
```

Comment:
This file should only import `TensorLib.Basic` which in turn will import `TensorLib.TensorElement` and `TensorLib.Tensor`